pypesto/optimize/result.py

```diff
@@ -79,18 +79,18 @@ def __init__(self,
                  message: str = None):
         super().__init__()
         self.id = id
-        self.x: np.ndarray = np.array(x) if x is not None else None
+        self.x: np.ndarray = np.asarray(x) if x is not None else None
```

Comment:
to be on the safe side results are not overwritten, I think I would enforce `np.array` in this function everywhere.